### DISCUSSION TITLE
Manage issues with labels, milestones and projects

### DISCUSSION THREAD

**Discussion by bjorn-ove** [community/community#3320] [Mobile]
Currently you can only open individual issues to set the various fielda. I have found no way to only view issues from one milestone or project to aid in re-prioritizing. Because of this I am only able to use the app when creating issues and have to use the mobile browser when I manage them. It would be great if it was possible to do directly in the app.

**Comment by eliperkins** [comment]
We hear ya! We'd love to see more ways within the app to view issues by label, milestones or by project too.

Stay tuned!
